[35m[Appium][39m Welcome to Appium v1.14.0
[35m[Appium][39m Non-default server args:
[35m[Appium][39m   port: 53620
[35m[Appium][39m   loglevel: info
[35m[Appium][39m   chromeDriverPort: 53621
[35m[Appium][39m   tmpDir: /var/folders/38/x4crs4qx0hn0r3_szny2t9sw0000gn/T//Katalon/Appium/Temp1565726266978
[35m[Appium][39m Appium REST http interface listener started on 0.0.0.0:53620
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m8 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"app":"/Users/sumittulsyan/Downloads/app-release (1).apk","noReset":false,"deviceName":"Xiaomi Redmi Note 4 (Android 7.0)","fullReset":true,"platform":"ANDROID","newCommandTimeout":1800,"platformVersion":"7.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"ffacc61e0004","systemPort":53619},"capabilities":{"firstMatch":[{"appium:app":"/Users/sumittulsyan/Downloads/app-release (1).apk","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","appium:deviceName":"Xiaomi Redmi Note 4 (Android 7.0)","appium:fullReset":true,"appium:newCommandTimeout":1800,"appium:noReset":false,"platform":"ANDROID","platformName":"android","appium:platformVersion":"7.0","appium:systemPort":53619,"appium:udid":"ffacc61e0004"}]}}[39m
[35m[BaseDriver][39m The capabilities ["platform"] are not standard capabilities and should have an extension prefix
[35m[Appium][39m Appium v1.14.0 creating new AndroidUiautomator2Driver (v1.33.1) session
[35m[Appium][39m Capabilities:
[35m[Appium][39m   platform: ANDROID
[35m[Appium][39m   platformName: android
[35m[Appium][39m   app: /Users/sumittulsyan/Downloads/app-release (1).apk
[35m[Appium][39m   autoGrantPermissions: true
[35m[Appium][39m   automationName: uiautomator2
[35m[Appium][39m   deviceName: Xiaomi Redmi Note 4 (Android 7.0)
[35m[Appium][39m   fullReset: true
[35m[Appium][39m   newCommandTimeout: 1800
[35m[Appium][39m   noReset: false
[35m[Appium][39m   platformVersion: 7.0
[35m[Appium][39m   systemPort: 53619
[35m[Appium][39m   udid: ffacc61e0004
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: 0b222327-5e72-4149-b3fd-0944eb8cadc1
[35m[BaseDriver][39m Using local app '/Users/sumittulsyan/Downloads/app-release (1).apk'
[35m[ADB][39m Found 1 'build-tools' folders under '/Users/sumittulsyan/.katalon/tools/android_sdk' (newest first):
[35m[ADB][39m     /Users/sumittulsyan/.katalon/tools/android_sdk/build-tools/29.0.1
[35m[ADB][39m Using 'adb' from '/Users/sumittulsyan/.katalon/tools/android_sdk/platform-tools/adb'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: ffacc61e0004
[35m[ADB][39m Using 'adb' from '/Users/sumittulsyan/.katalon/tools/android_sdk/platform-tools/adb'
[35m[ADB][39m Using 'apkanalyzer' from '/Users/sumittulsyan/.katalon/tools/android_sdk/tools/bin/apkanalyzer'
[35m[ADB][39m Using the alternative activity name detection method because of: Command '/Users/sumittulsyan/.katalon/tools/android_sdk/tools/bin/apkanalyzer -h manifest print '/Users/sumittulsyan/Downloads/app-release (1).apk'' exited with code 2
[35m[ADB][39m Using 'aapt' from '/Users/sumittulsyan/.katalon/tools/android_sdk/build-tools/29.0.1/aapt'
[35m[ADB][39m Extracting package and launch activity from manifest
[35m[ADB][39m Package name: 'io.nativebase.market.geekyants.taxiapprider'
[35m[ADB][39m Main activity name: 'io.nativebase.market.geekyants.taxiapprider.MainActivity'
[35m[ADB][39m Using 'apksigner' from '/Users/sumittulsyan/.katalon/tools/android_sdk/build-tools/29.0.1/apksigner'
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m io.nativebase.market.geekyants.taxiapprider was successfully uninstalled
[35m[ADB][39m The application at '/Users/sumittulsyan/Downloads/app-release (1).apk' is already cached to '/data/local/tmp/appium_cache/319121ffd2a4e89e1742e996fe8df850b3a56b9f.apk'
[35m[ADB][39m The installation of 'app-release (1).apk' took 3.128s
[35m[ADB][39m Error sending command, reconnecting device and retrying: shell,am,force-stop,io.appium.uiautomator2.server.test
[35m[UiAutomator2][39m Starting UIAutomator2 server 3.7.0
[35m[UiAutomator2][39m Using UIAutomator2 server from '/Users/sumittulsyan/Downloads/appium-mobile-test/node_modules/appium-uiautomator2-server/apks/appium-uiautomator2-server-v3.7.0.apk' and test from '/Users/sumittulsyan/Downloads/appium-mobile-test/node_modules/appium-uiautomator2-server/apks/appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[UiAutomator2][39m The instrumentation process has been unexpectedly terminated. Retrying UiAutomator2 startup (#1 of 1)
[35m[ADB][39m Error sending command, reconnecting device and retrying: shell,am,force-stop,io.appium.uiautomator2.server.test
[35m[ADB][39m Error sending command, reconnecting device and retrying: shell,pgrep,^uiautomator$
[35m[ADB][39m No 'uiautomator' process has been found
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: UnknownError: An unknown server-side error occurred while processing the command. Original error: Trying to proxy a session command without session id
[35m[ADB][39m Error sending command, reconnecting device and retrying: shell,am,force-stop,io.nativebase.market.geekyants.taxiapprider
[35m[UiAutomator2][39m Unable to force stop app: Error executing adbExec. Original error: 'Command '/Users/sumittulsyan/.katalon/tools/android_sdk/platform-tools/adb -P 5037 -s ffacc61e0004 shell am force-stop io.nativebase.market.geekyants.taxiapprider' exited with code 1'; Stderr: 'error: device 'ffacc61e0004' not found'; Code: '1'
[35m[ADB][39m Error sending command, reconnecting device and retrying: shell,dumpsys,package,io.nativebase.market.geekyants.taxiapprider
[35m[UiAutomator2][39m Unable to uninstall app: Error finding if 'io.nativebase.market.geekyants.taxiapprider' is installed. Original error: Error executing adbExec. Original error: 'Command '/Users/sumittulsyan/.katalon/tools/android_sdk/platform-tools/adb -P 5037 -s ffacc61e0004 shell dumpsys package io.nativebase.market.geekyants.taxiapprider' exited with code 1'; Stderr: 'error: device 'ffacc61e0004' not found'; Code: '1'
[35m[ADB][39m Error sending command, reconnecting device and retrying: forward,--remove,tcp:53619
[35m[UiAutomator2][39m Unable to remove port forward 'Error executing adbExec. Original error: 'Command '/Users/sumittulsyan/.katalon/tools/android_sdk/platform-tools/adb -P 5037 -s ffacc61e0004 forward --remove tcp\:53619' exited with code 1'; Stderr: 'adb: error: device 'ffacc61e0004' not found'; Code: '1''
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[31m500[39m [90m24231 ms - 819[39m
[35m[HTTP][39m [90m[39m
